tools/depends/target/Toolchain.cmake.in

```diff
@@ -33,26 +33,41 @@ elseif(OS STREQUAL osx)
 elseif(OS STREQUAL darwin_embedded)
   set(CMAKE_SYSTEM_NAME Darwin)
   set(CORE_SYSTEM_NAME darwin_embedded)
-  set(CORE_PLATFORM_NAME ios)
+  if(PLATFORM STREQUAL appletvos)
+    set(CORE_PLATFORM_NAME tvos)
+  else()
+    set(CORE_PLATFORM_NAME ios)
+  endif()
 endif()
 
 if(CORE_SYSTEM_NAME STREQUAL darwin_embedded)
   # Necessary to build the main Application (but not other dependencies)
   # with Xcode (and a bundle with Makefiles) (https://cmake.org/Bug/view.php?id=15329)
   if(NOT PROJECT_SOURCE_DIR MATCHES "tools/depends")
-    message(STATUS "Toolchain enabled IOS bundle for project ${PROJECT_NAME}")
     set(CMAKE_MACOSX_BUNDLE YES)
     set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED "NO")
-
-    # set this to YES once we have a deployment target of at least iOS 6.0
-    set(CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE "NO")
+    # Need to set this attribute to "" in order to
+    # complety disable code signing
+    # see: https://gitlab.kitware.com/cmake/cmake/issues/19112
+    set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "")
+    message(STATUS "Toolchain enabled ${CORE_PLATFORM_NAME} bundle for project ${CORE_PLATFORM_NAME}")
+    if(CORE_PLATFORM_NAME STREQUAL tvos)
+      set(CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE "YES")
+    else()
+      # set this to YES once we have a deployment target of at least iOS 6.0
+      set(CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE "NO")
+    endif()
   endif()
 endif()
 
 if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
   set(CMAKE_OSX_SYSROOT @use_sdk_path@)
   set(CMAKE_XCODE_ATTRIBUTE_ARCHS ${CPU})
-  set(CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET 9.0)
+  if(CORE_PLATFORM_NAME STREQUAL tvos)
+    set(CMAKE_XCODE_ATTRIBUTE_TVOS_DEPLOYMENT_TARGET 11.0)
+  elseif(CORE_PLATFORM_NAME STREQUAL ios)
+    set(CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET 9.0)
```

Comment:
Is the if needed?
does this cause issues if `CMAKE_XCODE_ATTRIBUTE_TVOS_DEPLOYMENT_TARGET` is set for an iOS build and `CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET` for a tvOS build?

Reply:
More just why add attributes that aren't needed to a project. May handle fine now, but who's to say what will happen in the future. Why not just add what's needed as needed